inspirehep/alembic/f5bf1193b078_recreate_workflows_record_sources_table.py

```diff
@@ -0,0 +1,96 @@
+#
```

Comment:
Was this migration AUTOgenerated?
http://alembic.zzzcomputing.com/en/latest/autogenerate.html
Please do NOT deploy this now, but align with myself or Micha before, as we are working on fixing the migration history and the schema differences between prod db and qa db.